app/routes.py

```diff
@@ -184,6 +184,35 @@ def check_out():
     cart=CartItem.query.filter_by(user_id=user_id).first()
     if cart:
         cart.remove()
-        return jsonify({'message':'Your Order Placed Successfully and Your order is will delieverd to your registered address'+Address})
+        product = Product.query.filter_by(id=product_id).first()
+        if product:
+            retailer = User.query.filter_by(id=user_id).first()
+            if retailer:
+                msg = Message(subject='New Order', sender=os.getenv('MAIL_USERNAME'), recipients=[retailer.email])
+                msg.body = f"Dear {retailer.username}, you have a new order for the product: {product.product_name}."
+                mail.send(msg)
+                notifi=f"Dear {retailer.username}, you have a new order for the product: {product.product_name}."
+                new_notification=Notification(user_id=user_id,product_id=product_id,Notifications=notifi) 
+                new_notification.add()       
+            return jsonify({'message':'Your Order Placed Successfully and Your order is will delieverd to your registered address'+Address})
+        
     else:
         return jsonify({'message':'cart is empty'})
+
+@app.route('/notification',methods=['GET','POST'])
+@is_retailer
+def notification():
+    data=request.get_json()
+    username=data['username']
+    password=data['password']
+    user_id=data['user_id']
+    user=User.query.filter_by(username=username).first()
+    notify=Notification.query.all()
+    qrr=[]
+    for notified in notify:
```

Comment:
remove loop.
Filter notifications via user_id.
Latest notifications comes on the top.